lib/ruby_home/config/services.yml

```diff
@@ -382,6 +382,24 @@
   - 00000023-0000-1000-8000-0026BB765291
   :required_characteristics_uuids:
   - 00000025-0000-1000-8000-0026BB765291
+- :name: :television
+  :description: Television
+  :uuid: '000000D8-0000-1000-8000-0026BB765291'
+  :optional_characteristics_uuids:
+  - 00000008-0000-1000-8000-0026BB765291
+  - 000000DD-0000-1000-8000-0026BB765291
+  - 00000136-0000-1000-8000-0026BB765291
+  - 000000E0-0000-1000-8000-0026BB765291
+  - 00000137-0000-1000-8000-0026BB765291
+  - 000000E2-0000-1000-8000-0026BB765291
+  - 000000DF-0000-1000-8000-0026BB765291
+  - 00000023-0000-1000-8000-0026BB765291
```

Comment:
Do these need to be in any particular order?

Reply:
No, the order of these uuids doesn't matter.